ragna/deploy/_ui/central_view.py

```diff
@@ -389,11 +396,16 @@ async def chat_callback(
                 user=self.get_user_from_role("assistant"),
                 sources=answer["sources"],
                 on_click_source_info_callback=self.on_click_source_info_wrapper,
+                toolbar_visible=False,
             )
             yield message
 
+            content = answer["content"]
             async for chunk in answer_stream:
                 message.content_pane.object += chunk["content"]
+                content += chunk["content"]
```

Comment:
Instead doing a second repeated string concatenation here, can't we just do `message.clipboard_button.value = message.content_pane.object` after the loop?